spring-cloud-stream-app-dependencies/pom.xml

```diff
@@ -561,6 +568,11 @@
 				<version>${pmml.version}</version>
 			</dependency>
 			<dependency>
+				<groupId>javax.mail</groupId>
+				<artifactId>mail</artifactId>
+				<version>1.5.0-b01</version>
```

Comment:
M-m-m. Not sure where my comment from here has gone.
But should not use `javax.mail` artifact.
Only `com.sun.mail:javax.mail`. 
And rely already on the Boot inheritance. 
Therefore remove this one.